values.yaml

```diff
@@ -137,6 +137,15 @@ global:
       limits: {}
       requests: {}
 
+  # External ES logging
+  custom_logging:
+    enabled: false
+    scheme: https
+    host: ""
+    port: ""
+    secret: ""
+    #secretName: ~
```

Comment:
@danielhoherd - vishwas asked to keep something like secretKey instead of secretName.

Reply:
What we call it depends on what it is. If you mean "key" as in `key: value` that's probably not the right term. If you mean "key" as in "cryptographic key" then yeah, that works.

Reply:
what about `token` or `credentials`? do either of those fit better?

Reply:
Did we ever make a decision here?

Reply:
yes - we are going with below formats
```bash
secret: "" - for base64 encode 
secretName: ~ - to provide k8s secret
awsSecretName: ~ - for aws credentials
awsIAMRole: ~ - for deployment/namespace level annotation 
awsServiceAccountAnnotation: ~ - for IRSA use case 
```

Reply:
So the comments are correct? If so resolve this thread.